### PR TITLE
feat: mobile-qa-automation

### PR DESCRIPTION
**Pull request by PrinceSoni83**
### Changes
- Add Readme for Mobile QA Automation
- Update Readme for Web QA Automation

### CODE REVIEW

**Review by Elena0420 — DISMISSED** [review]

File: mobile-app-automation/readme.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+1. Include all tests related document in same github repository.
+
+- Use below package and activity name (Appium)
+  - appPackage=com.amazon.mShop.android.shopping
+  - appActivity=com.amazon.mShop.home.HomeActivity
+
+## Acceptance Criteria
+
+- Scenario 1: [Check Add a new Payment to your profile]
+   1. Starts on the [Amazon](https://www.amazon.com/)
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:24]: Prince, please remove the URL and add `Region: United State` instead

File: mobile-app-automation/readme.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+   2. Tap on Gift Cards
+   3. Check the balance of your gift card is $0.00
+
+
+- Scenario 3: [Check the total displayed number of results for category Smart Home | Televisions]
+   1. Go to Categories list
+   2. Choose category 'Smart Home'
+   3. Choose sub-category category 'Home Entertainment'
+   4. Choose sub-category 'Televisions'
+   5. Check the total number of results match the total displayed products
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md]: I believe, all steps needs to be revisit as mobile app doesn't have the same like web, thanks

Comment (PrinceSoni83) [reply mobile-app-automation/readme.md]: @MohmedEmad I added it as is, only after when I received the tasks after internal review among the QA team.

@Elena0420  can you please give us the exact steps as I tried to implement what I have received.

**Review by Remon-Refaat — DISMISSED** [review]

File: mobile-app-automation/readme.md (hunk 2 above)
Comment (MohmedEmad) [reply mobile-app-automation/readme.md]: @PrinceSoni83 I will provide you with the detailed steps Today, thanks

File: mobile-app-automation/readme.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+1. Prepare a test summary report and include the information about platform, app, tools etc
+1. Include all tests related document in same github repository.
+
+- Use below package and activity name (Appium)
+  - appPackage=com.amazon.mShop.android.shopping
+  - appActivity=com.amazon.mShop.home.HomeActivity
+
+## Acceptance Criteria
+
+- Scenario 1: [Check Add a new Payment to your profile]
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:23]: 1. Open Amazon app with region United State
2. Login with valid credentials
3. Tap on Your account
4. From Payments section, select Your Payments
5. Tap on Wallet and add a credit or debit card as the payment method
6. Assert on added card with correct type [Visa/Mastercard]

File: mobile-app-automation/readme.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+   1. Starts on the [Amazon](https://www.amazon.com/)
+   2. Login with valid credentials
+   3. Tap on Your account
+   4. Tap on the Payment option
+   5. Tap on Wallet and add a credit or debit card as the payment method
+
+
+- Scenario 2: [Check your gift card's balance is $0.00]
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:32]: 1. Tap on Your account
2. From Payments section, select Manage gift card balance 
3. Check the balance of your gift card is $0.00

File: mobile-app-automation/readme.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+   1. Tap on Your account
+   2. Tap on Gift Cards
+   3. Check the balance of your gift card is $0.00
+
+
+- Scenario 3: [Check the total displayed number of results for category Smart Home | Televisions]
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:38]: 1. Click on Shop by Department from burger menu
2. Choose category ‘Electronics’
3. Choose sub-category category ’TV & VIDEO’
4. Filter with ’Smart TV’ and click show results
5. Check the total number of results match the total displayed in filter

File: mobile-app-automation/readme.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+   1. Go to Categories list
+   2. Choose category 'Smart Home'
+   3. Choose sub-category category 'Home Entertainment'
+   4. Choose sub-category 'Televisions'
+   5. Check the total number of results match the total displayed products
+
+- Scenario 4: [Check Add a new Address to your profile]
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:45]: 1. Tap on Your account
2. From Account settings, select Your Address
3. Add a new address
4. Assert on added address under personal Addresses

File: mobile-app-automation/readme.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    1. Tap on Your account
+    2. Tap on Manage address book
+    3. Add new address
+    4. Verify if the address is added
+
+- Scenario 5: [Check the selected currency displayed for the products' price]
```
Comment (MohmedEmad) [inline mobile-app-automation/readme.md:51]: 1. Go to Settings
2. Select Country & Language
3. Change currency from 'USD' to 'AED'
3. Save changes
4. Assert on selected currency displayed for Deals and Promotions products

**Review by MohmedEmad — COMMENTED** [review]
@PrinceSoni83 I have added all the steps for 5 test scenarios, thanks

**Review by MohmedEmad — APPROVED** [review]

**Pull request merged**